tests/unit_tests/commandline/test_geoips_list_interfaces.py

```diff
@@ -9,24 +9,48 @@
 
 
 class TestGeoipsListInterfaces(BaseCliTest):
-    """Unit Testing Class for List Interfaces Sub-Command."""
+    """Unit Testing Class for List Interfaces Command."""
 
     @property
-    def all_possible_subcommand_combinations(self):
+    def command_combinations(self):
         """A list of every possible call signature for the GeoipsListInterfaces command.
 
         This includes failing cases as well.
         """
         if not hasattr(self, "_cmd_list"):
-            self._cmd_list = [self._list_interfaces_args]
+            base_args = self._list_interfaces_args
+            self._cmd_list = [base_args]
             for pkg_name in self.plugin_package_names:
-                self._cmd_list.append(
-                    self._list_interfaces_args + ["-i", "-p", pkg_name]
-                )
-            # Add argument list which invokes the help message for this command
-            self._cmd_list.append(["geoips", "list", "interfaces", "-h"])
+                self._cmd_list.append(base_args + ["-i", "-p", pkg_name])
+            # Add argument list which selects certain columns for generic interfaces
+            self._cmd_list.append(
+                base_args
+                + [
+                    "--columns",
+                    "package",
+                    "interface",
+                    "supported_famlies",
+                    "abspath",
+                ]
+            )
+            # Add argument list which selects certain columns for implemented interfaces
+            self._cmd_list.append(
+                base_args
+                + [
+                    "-i",
+                    "-p",
+                    "geoips",
+                    "--columns",
+                    "interface",
+                    "plugin_type",
+                ]
+            )
             # Add argument list with an invalid command call ("-p" w/out "-i")
```

Comment:
Is this comment still correct?

Reply:
No it's not. Nice catch and I can remove that.

Reply:
This has been removed BTW.